src/pages/TasksPage.js

```diff
@@ -58,14 +58,17 @@ function TasksPage() {
     setCurrentTask(e.target.value);
   };
 
-  const handleAdd = (event) => {
+  const handleAdd = async (event) => {
     event.preventDefault();
     if (currentTask) {
-      setTaskList([...taskList, currentTask]);
-      setCurrentTask("");
-    } else {
-      alert("Please enter a To Do...");
+      const docRef = await addDoc(collection(db, "tasks"), {
+        task: currentTask,
+      });
     }
+
+    setCurrentTask("");
+    const newTask = { task: currentTask };
```

Comment:
`docRef.id` ile id'yi de eklemezsek tasklar listelenirken hata almaz mıyız?

Reply:
Hayır hata almayız ama güncelleme silme gibi durumları id ye ihtiyacımız olduğu için yapamayız